source/manuals/programming-languages/html.html.md.erb

```diff
@@ -1,85 +1,153 @@
+for this.
 
-## Main content
+### Sectioned content
 
-The `<main>` tag should be used to identify the main content of your page. Support for it is still marginal so ensure to add the `role="main"` attribute.
+Use the `<section>` tag to represent standalone, thematic groupings of content
+where a more specific semantic element would not be appropriate.
 
-## Sectioned content
+Examples of sections would be chapters, or various tabbed pages in a tabbed dialog box,
+or a home page split into sections such as introduction, news items and contact information.
 
-When using the `<section>` tag, add an `aria-labelledby` attribute linking to the id of the heading that identifies that section.
+You should add an `aria-labelledby` attribute linking to the id of the heading
+that identifies that section.
 
 Example:
 
-    <section aria-labelledby="advice">
-      <h1 id="advice">Help and advice</h1>
+```html
+<section aria-labelledby="introduction">
+    <h1 id="introduction">Introduction</h1>
+    <p>People have been catching fish for food since before recorded history...</p>
+</section>
 
-## Navigational elements
+<section aria-labelledby="equipment">
+    <h1 id="equipment">Equipment</h1>
+    <p>The first thing you'll need is a fishing rod or pole that you find comfortable
+    and is strong enough for the kind of fish you're expecting to try to land...</p>
+</section>
+```
 
-When using the `<nav>` tag, add the ARIA landmark `role="navigation"`.
+### Navigational elements
 
-The detail of using the `<nav>` tag is covered in this [blog post](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
+Use `<nav role="navigation">` to wrap groups of links that aren't already in
+another context (e.g. `<footer>`). Common examples of navigation sections are
+menus, tables of contents, and indexes.
 
-### Emphasis
+As with `<section>`, it is advisable to use `aria-labelledby` to label your `<nav>`.
+This helps to distinguish different `<nav>` blocks in the page.
 
-If you want to give the current location emphasis within a navigational list, use a `<strong>` tag.
+The GOV.UK blog has [more information regarding the use of the `<nav>` tag](https://insidegovuk.blog.gov.uk/2013/07/03/rethinking-navigation/).
 
-Example:
+### Aside
+
+`<aside role="complementary">` should be used for content related to the primary content
+of the webpage, but which does not constitute the primary content of the page.
+
+Examples include author information, related links and related content.
+
+### Footer
+
+`<footer role="contentinfo">` should be used for the footer of the site.
+
+## Individual element guidance
+
+### Headings
+
+Headings should be in order - for example `<h1>` then `<h2>`, but not `<h1>` then `<h3>`.
+
+There should only be one `<h1>` element in a page.
+
+### Text emphasis
+
+Use `<strong>` and `<em>` when styling text that should be read with _emphasis_.
+Use of `<b>` and `<i>` is acceptable if used purely for visual styling.
```

Comment:
I don't agree with this - if you're using a tag for visual styling only on a word or phrase, then it should be done with a span element and a specific class.

https://developer.mozilla.org/en-US/docs/Web/HTML/Element/b

Reply:
The example in that doc suggests using it for drawing visual attention to individual keywords, e.g.

> The two most popular science courses offered by the school are `<b class="term">`chemistry`</b>` (the study of chemicals and the composition of substances) and `<b class="term">`physics`</b>` (the study of the nature and properties of matter and energy).

But as you say, discourages using `<b>` for general styling:

> However, you should not use `<b>` for styling text; instead, you should use the CSS `font-weight` property to create boldface text, or the `<strong>` element to indicate that text is of special importance.

I should rephrase the sentence to make it clear that it's only about drawing visual attention to individual words 👍

Reply:
Have gone down a rabbit hole re-educating myself about `<strong>` and `<em>` and `<b>` and `<i>` and have boiled it down to this:

```md
You should avoid bold or italics for blocks of text as it can make it more
difficult to read.

Use `<em>` when styling words that should be read with _emphasis_. The emphasised
words is read in a different tone of voice by screen readers. Browsers traditionally
render English `<em>` text in italics, but other languages may denote emphasis
differently.

Use `<strong>` for content of strong importance or urgency (such as warnings). Again,
`<strong>` text is read in a different tone of voice by screen readers, and again,
browsers traditionally render English `<strong>` text in a bold style, but other
languages may denote importance differently.

As a general rule, avoid using `<i>` or `<b>`. They are purely to apply visual
styling without additional meaning, which is better achieved using the `font-style`
and `font-weight` CSS properties. Keeping presentation information separate from
your HTML content is considered best practice.
```